### ISSUE TITLE
Kubernetes is not showing in Settings when using Docker Edge

### ISSUE THREAD

**Issue by ctmcisco** [docker/for-win#1747]
### Expected behavior
Install Docker Edge and have Kubernetes server cluster and client installed 

### Actual behavior
only have Kubernetes client / CLI 

![image](https://user-images.githubusercontent.com/14852041/36631799-9c8ef494-1943-11e8-8f7e-018974ca9450.png)

 Kubernetes option in settings is missing

### Information

A diagnostic was uploaded with id: 8D651FE0-4FDA-4821-92F3-3C01E7ED516E/2018-02-24_09-10-07

![image](https://user-images.githubusercontent.com/14852041/36631781-69ab0aae-1943-11e8-9dae-1c852977244b.png)

### Steps to reproduce the behavior

  1. Right click on docker icon
  2. Select Settings
  3. No Kubernetes option

![image](https://user-images.githubusercontent.com/14852041/36631756-195265ac-1943-11e8-91c3-67edbccf6503.png)

**Comment by carlfischer1** [comment]
@ctmcisco at this time Kubernetes support is only available when running in Linux containers mode. Use the whale icon in the system tray to select Linux containers.

**Comment by ctmcisco** [comment]
I have everything working now. I found some some issues with Docker for Windows starting up when Windows starts. 

I disabled that setting and seems to allow the Container and Docker services to start and prepare before DFW starts. 

I was able to now swap from Windows to Linux containers and get Kubernetes installed.



![image](https://user-images.githubusercontent.com/14852041/37048141-9bd2eee2-2132-11e8-9abb-9be3fcc57471.png)

I am barely scratching the surface on kubernetes and if you are looking for the Kubernetes web portal will recommend looking this gentleman: 

 Scott Hanselman's 
(http://twitter.com/shanselman) 

How to set up Kubernetes on Windows 10 with Docker for Windows and run ASP.NET Core

https://www.hanselman.com/blog/HowToSetUpKubernetesOnWindows10WithDockerForWindowsAndRunASPNETCore.aspx
